Extras/HistoryView/UIKit/COTrackViewController.m

```diff
@@ -107,7 +107,7 @@ - (UIColor *)suggestedColorForNode: (id <COTrackNode>)node
 
 - (UITableViewCell *)makeCellForNode: (id <COTrackNode>)node
 {
-    static NSString *TrackCellIdentifier = @"TrackCell";
+    static const NSString *const TrackCellIdentifier = @"TrackCell";
```

Comment:
TrackCellIdentifier should be declared in a way that matches what we do for global string constants. So I would declare it:

`static NSString *const TrackCellIdentifier = @"TrackCell"`

iirc FOUNDATION_EXPORT is mapped to `extern` but not `extern const`.